.github/dependabot.yml

```diff
@@ -1,11 +1,11 @@
-version: 2
-updates:
-  - package-ecosystem: "npm"
-    directory: "/"
-    schedule:
-      interval: "weekly"
-
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "weekly"
\ No newline at end of file
+#version: 2
```

Comment:
Let's remove the file altogether. There is no reason for commenting it out. We can always get it from git history if needed